File: SistemaDecomisos/src/java/model/Persona.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

/**
 *
 * @author Marco
 */
public class Persona {
    protected String identificacion;
    protected String nombre;
    protected String apellido1;
    protected String apellido2;

    public Persona() {
    }

    public Persona(String identificacion, String nombre, String apellido1, String apellido2) {
        this.identificacion = identificacion;
        this.nombre = nombre;
        this.apellido1 = apellido1;
        this.apellido2 = apellido2;
    }

    public String getIdentificacion() {
        return identificacion;
    }

    public void setIdentificacion(String identificacion) {
        this.identificacion = identificacion;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido1() {
        return apellido1;
    }

    public void setApellido1(String apellido1) {
        this.apellido1 = apellido1;
    }

    public String getApellido2() {
        return apellido2;
    }

    public void setApellido2(String apellido2) {
        this.apellido2 = apellido2;
    }
    
}

File: SistemaDecomisos/web/js/objects/Persona.js
function Persona(identificacion, nombre, apellido1, apellido2, direccionExacta) {
    this.Persona(identificacion, nombre, apellido1, apellido2, direccionExacta);
}

Persona.prototype = {
    identificacion : "", 
    nombre : "", 
    apellido1 : "",
    apellido2 : "", 
    direccionExacta : "",
    Persona: function (identificacion, nombre, apellido1, apellido2, direccionExacta) {
        this.identificacion = identificacion, 
        this.nombre = nombre;
        this.apellido1 = apellido1;
        this.apellido2 = apellido2;
        this.direccionExacta = direccionExacta;
    }
};

Persona.from = function (plain) {
    persona = new Persona(
        plain.identificacion, 
        plain.nombre, 
        plain.apellido1, 
        plain.apellido2, 
        plain.direccionExacta
    );
    return persona;
};

Persona.to = function (persona) {
    return {
        _class: 'Persona',
        identificacion : persona.identificacion, 
        nombre : persona.nombre, 
        apellido1 : persona.apellido1, 
        apellido2 : persona.apellido2, 
        direccionExacta : persona.direccionExacta
    };
};
 